src/Kmd.Logic.Audit.Client.SerilogSeq/SerilogSeqAuditClientConfigurationExtensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Net.Http;
+using Serilog;
+
+namespace Kmd.Logic.Audit.Client.SerilogSeq
+{
+    public static class SerilogSeqAuditClientConfigurationExtensions
+    {
+        public static LoggerConfiguration DefaultConfiguration(this SerilogSeqAuditClientConfiguration config, HttpMessageHandler messageHandler = null)
```

Comment:
I think we should use a verb in this method name.

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-methods

We are not the framework, but most people follow this idea.

```suggestion
        public static LoggerConfiguration CreateDefaultConfiguration(this SerilogSeqAuditClientConfiguration config, HttpMessageHandler messageHandler = null)
```

Reply:
Agreed. Changed